Logos/agari_logo.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="LogoFullColorRGB" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
+	 y="0px" viewBox="0 0 393 211.1" style="enable-background:new 0 0 393 211.1;" xml:space="preserve">
+<style type="text/css">

```

Comment:
Logo file should not contain style tag Please refer the checklist for logo validation mention below:


Ensure raw file of logo does not have any of the following:
--
cls and style formats
embedded png formats
xmlns:xlink
data-name
xlink:href is not there and inline is used instead
title tag is not used
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then these IDs are set as GUIDs so that these are uniquely identifiable across all Azure logo assets
Logo scales well to fit in a 75 px square

Thanks

Reply:
@udinagari  File still have the style tag , xmlns:xlink  attribute and id is not a guid.

Please refer the guidelines here  https://github.com/Azure/Azure-Sentinel/blob/e1665d4c4442b430d6847adc65f4b9c25f7b071f/DataConnectors/ReadMe.md.

Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name
xlink:href is not there and inline is used instead
title tag is not used
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then these IDs are set as GUIDs so that these are uniquely identifiable across all Azure logo assets
Logo scales well to fit in a 75 px square

Thanks